Cluster/Connecting to secure clusters with PowerShell.md

```diff
@@ -39,17 +38,19 @@ $ClusterName= "{yourclustername}.{region}.cloudapp.azure.com:19000"
 
 #single command - compact example
 $connectArgs = @{  ConnectionEndpoint = $ClusterName;   }
-Connect-ServiceFabricCluster @connectArgs 
+Connect-ServiceFabricCluster @connectArgs
 ```
 
 Simple AAD Authentication
+
 ```PowerShell
```

Comment:
The language name for code snippets in mark down should be in lower case. But looks like there is no official documentation and it is flexible interpreted.

Reply:
I am fixing any changes proposed by a VSCode plugin to the MD file to maintain uniformity. I will check if there are any warnings related to casing

Reply:
fixed